katsdpingest/katsdpingest/sigproc.py

```diff
@@ -206,7 +333,8 @@ class CountFlagsTemplate:
 
         - wgs: number of workitems per workgroup in channel dimension
     """
-    autotune_version = 1
+
+    autotune_version = 2
```

Comment:
Why a new version? I'm guessing to invalidate older cached versions.

Reply:
Exactly.